### ISSUE TITLE
Cannot type text when switching back to MM

### ISSUE THREAD

**Issue by breki** [RickStrahl/MarkdownMonster#468]
When I switch away from MarkdownMonster and then switch back (either using the mouse or using Alt+Tab, I cannot enter text into the editor (it just ignores keystrokes). What is weird is that the cursor caret is still shown and I can move it using the cursor keys (left, right, top, down). I can also delete the text (both with backspace and del), but any alphanumeric keys are ignored. I have to use the mouse to click into the editor to be able to enter text.

**Comment by RickStrahl** [comment]
Yeah seen too. There have been some focus changes in the editor, but that behavior is not desired. There is code there to re-activate the editor when focus returns to the window (always to the editor) - gotta check why that's not working right. I suspect a timing issue of some sort.

**Comment by RickStrahl** [comment]
Ok, so I've fixed editor focus and also properly handle focus management to reset focus to the active control. If something other than the editor is open focus will stay with that control. The old (pre-breakage) behavior was to **always** focus the editor even if something else was focused. The new behavior leaves whatever is focused in focus.

aff6ecded83870ab46a03cd4c05aaf45b2ba1e15

Updated in 1.14.5
